data/openstudio_standards_data/database_files/level_2_lighting_space_types.json

```diff
@@ -0,0 +1,5666 @@
+[
+    {
+        "lighting_space_type_name": "airport_concourse_lighting",
+        "level_3_lighting_code_definition_table": "level_3_lighting_90_1_2004",
```

Comment:
what is the difference between level 2 and level 3?  could you add that descriptor to the documentation?

Reply:
It's already included in the documentation:

> - The second level, level 2, is a collection of data tables that contains lists of sub-space type names and references to different code requirement values.
> - The third and final level, level 3, is a collection of data tables that contains raw information from building energy codes, such as lighting power allowances and required ventilation rates. The list of space type provided in the level 1 table is a concise yet exhaustive list of space type that can be used to model most commercial building spaces. Assumptions for these space types are contained in the level 2 and level 3 tables, however these tables contain additional information about more specific space type requirements (or less specific, e.g., through the building area method lighting power allowances).